app/backend/src/couchers/db.py

```diff
@@ -13,6 +15,13 @@
 logger = logging.getLogger(__name__)
 
 
+def apply_migrations():
+    alembic_cfg = Config("alembic.ini")
```

Comment:
Please make the code independent of current working directory, by using `__file__`.

Reply:
I can try, but I'll have to investigate how alembic does paths: it needs access to the migrations files

Reply:
A temporary chdir solved both problems; I've pushed to your branch :)